content/java/concepts/array-list/terms/replaceAll/replaceAll.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.replaceAll()'
+Description: 'Replaces all occurrences of a given value ArrayList.'
+Subjects:
+  - 'A subject name'
+  - 'A second subject name'
+  - 'An nth subject name'
+Tags:
+  - 'Computer Science'
+  - 'Information Technology'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.replaceAll()`** method of the `ArrayList` class takes an Operator as argument and applies it to every element in the List
```

Comment:
```suggestion
The `.replaceAll()` method of the `ArrayList` class takes an operator as argument and applies it to every element in the list
```

Reply:
Should we not keep the first replaceAll() occurence bold as per this guide? https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#writing-entries.
But maybe remove the backtick?

Reply:
@cigar-galaxy82 See my questsion above ☝️